xc/xenops_server_xen.ml

```diff
@@ -2160,6 +2160,35 @@ module VIF = struct
 			_locking_mode, "disabled";
 		]
 
+	let xenstore_of_static_ip_setting vif =
+		let constant_setting = [ "static-ip-setting/mac", vif.mac;
+					 "static-ip-setting/error-code", "0";
+					 "static-ip-setting/error-msg", "" ]
+		in
+		let ipv4_setting = match vif.ipv4_configuration with
+		| Vif.Unspecified4 -> [ "static-ip-setting/enabled", "0" ]
```

Comment:
You don't need to use the `Vif.` prefix here (and below), because there is an `open Vif` above.